apps/src/tutorialExplorer/filterGroup.jsx

```diff
@@ -23,12 +24,15 @@ const FilterGroup = React.createClass({
     text: React.PropTypes.string.isRequired,
     filterEntries: React.PropTypes.array.isRequired,
     selection: React.PropTypes.array.isRequired,
-    onUserInput: React.PropTypes.func.isRequired
+    onUserInput: React.PropTypes.func.isRequired,
+    windowWidth: React.PropTypes.number.isRequired
   },
 
   render() {
+    var filterGroupOuterStyle = {};
+    Object.assign(filterGroupOuterStyle, styles.filterGroupOuter, {float: "left", width: getItemWidthMobile(50, this.props.windowWidth)});
```

Comment:
note this can just be:
```
const filterGroupOuterStyle = Object.assign({}, styles.filterGroupOuter, { float: 'left', width: 'getItemWidthMobile(50, this.props.windowWidth);`
```
or if you want to be even more ES6y
```
const filterGroupOuterStyle = {
  ...styles.filterGroupOuter,
  float: 'left',
  width: getItemWidthMobile(50, this.props.windowWidth)
}